test/integration/running/test_running.py

```diff
@@ -476,3 +476,47 @@ def test_no_args_usage(tt_cmd):
             # Check that the process was terminated correctly.
             instance_process_rc = instance_process.wait(1)
             assert instance_process_rc == 0
+
+
+def test_running_env_variables(tt_cmd, tmpdir_with_cfg):
+    tmpdir = tmpdir_with_cfg
+    # Copy the test application to the "run" directory.
+    test_app_path = os.path.join(os.path.dirname(__file__), "test_env_app", "test_env_app.lua")
+    shutil.copy(test_app_path, tmpdir)
+    # Set environmental variable which redirects logs into file.
+    my_env = os.environ.copy()
+    my_env["TT_LOG"] = "file:"+os.path.join(str(tmpdir), "tarantool_env.log")
```

Comment:
I think that using `log` to check functionality is not a good idea because it can be filtered out in the future in https://github.com/tarantool/tt/issues/293

Reply:
Changed